adding_species_manuscript.tex

```diff
@@ -516,7 +514,7 @@ \subsection*{Basic setup for chromosome-level simulations}
   A given species might have more than one demographic model, fit from different data or by different methods.
   Thus, when selecting a demographic model, one should examine the data sources and methods used to obtain it to ensure that they are relevant to their study.
   At a minimum, simulation requires a single estimate of \textbf{effective population size}. This estimate, which may correspond to some sort of historical average effective population size,
-  should reproduce in simulation the average observed genetic diversity in that species. Note, however, that this average effective population size will not capture features of genetic variation that are caused by recent changes in population size and the presence of population structure \citep{MacLeod2013,Eldon2015}.
+  should translate in simulation to the average observed genetic diversity in that species. Note, however, that this average effective population size will not capture features of genetic variation that are caused by recent changes in population size and the presence of population structure \citep{MacLeod2013,Eldon2015}.
```

Comment:
I prefer the original wording here, since 'translate' is a bit vague, whereas 'reproduce' means that the simulated data will have the same average genetic diversity as observed in real genomes.